pkg/fanal/analyzer/language/conda/environment/environment.go

```diff
@@ -2,32 +2,108 @@ package environment
 
 import (
 	"context"
+	"fmt"
 	"os"
 	"path/filepath"
+	"sync"
 
+	"github.com/bmatcuk/doublestar/v4"
 	"golang.org/x/xerrors"
 
 	"github.com/aquasecurity/trivy/pkg/dependency/parser/conda/environment"
+	"github.com/aquasecurity/trivy/pkg/dependency/parser/conda/meta"
 	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
 	"github.com/aquasecurity/trivy/pkg/fanal/analyzer/language"
 	"github.com/aquasecurity/trivy/pkg/fanal/types"
+	"github.com/aquasecurity/trivy/pkg/log"
+	xio "github.com/aquasecurity/trivy/pkg/x/io"
 )
 
 func init() {
 	analyzer.RegisterAnalyzer(&environmentAnalyzer{})
 }
 
-const version = 1
+const version = 2
+
+type parser struct{}
+
+func (*parser) Parse(r xio.ReadSeekerAt) ([]types.Package, []types.Dependency, error) {
+	p := environment.NewParser()
+	pkgs, err := p.Parse(r)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	once := sync.Once{}
+	for i, pkg := range pkgs.Packages {
+		// Skip packages without a version, because in this case we will not be able to get the correct file name.
+		if pkg.Version != "" {
+			licenses, err := findLicenseFromEnvDir(pkg, pkgs.Prefix)
+			if err != nil {
+				// Show log once per file
+				once.Do(func() {
+					log.WithPrefix("conda").Debug("License not found. See https://aquasecurity.github.io/trivy/latest/docs/coverage/os/conda/#license_1 for details",
```

Comment:
You can use this function for a versioned document link.
https://github.com/aquasecurity/trivy/blob/3d4ae8b5be94cd9b00badeece8d86c2258b2cd90/pkg/version/doc/doc.go#L25-L31

```suggestion
					log.WithPrefix("conda").Debugf("License not found. See %s for details", doc.URL("docs/coverage/os/conda/", "license_1"))
```

Reply:
Oh, sorry, I missed some arguments, and my suggestion was broken.

>log.String("pkg", pkg.Name), log.Err(err))

Reply:
i am fixing that